### PR TITLE
[TS-158]: Recent Activities test implemented

### PR DESCRIPTION
**Pull request by GadoliniumPotassium**

### CODE REVIEW

**Review by rileykc98 — COMMENTED** [review]
Looks good! We'll wait and make sure the tests pass. In the meantime, we've been using "-" naming conventions instead of camel case for data-cy props. Could you change your code to match?

File: src/pages/Dashboard/Dashboard.vue [hunk 1]
```diff
@@ -53,10 +53,10 @@
               :today="activity.today"
             ></Recents>
             <router-link
-              to="/news-feed"
+              to="/tradezone"
```
Comment (rileykc98) [inline src/pages/Dashboard/Dashboard.vue:56]: Good catch on the outdated link!

Comment (GadoliniumPotassium) [reply src/pages/Dashboard/Dashboard.vue:56]: thanks king :crown:

**Review by rileykc98 — DISMISSED** [review]
Good to go, thanks for updating the naming conventions!

File: src/components/TradeZone/Feed.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <v-container class="pa-0 ma-0">
+    <v-container class="pa-0 ma-0" data-cy="tradezoneFeed">
```
Comment (itsShaVawn) [inline src/components/TradeZone/Feed.vue:2]:
```suggestion
    <v-container class="pa-0 ma-0" data-cy="trade-zone-feed">
```

Comment (itsShaVawn) [reply src/components/TradeZone/Feed.vue:2]: just to follow the - naming convention!

**Review by itsShaVawn — APPROVED** [review]
Check my comments, but other than that: great work 👌🏼

**Review by itsShaVawn — APPROVED** [review]
Nice work!

**Review by Ming424 — APPROVED** [review]
good job, looks good to me 😎

**Review by Ming424 — APPROVED** [review]
approved, good job!

File: src/components/TradeZone/Feed.vue [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <template>
-  <v-container fluid>
+
```
Comment (itsShaVawn) [inline src/components/TradeZone/Feed.vue:2]: You'll have to remove this extra space

**Review by itsShaVawn — APPROVED** [review]

File: src/components/TradeZone/Feed.vue (hunk 3 above)
Comment (rileykc98) [reply src/components/TradeZone/Feed.vue]: @GadoliniumPotassium Hope you don't mind I went ahead and did that for you. Re-requesting reviews now

**Review by itsShaVawn — APPROVED** [review]

**Review by rileykc98 — APPROVED** [review]

**Pull request merged**